.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,3 +1,12 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
```

Comment:
Some of the text below makes the assumption you could be doing something beyond a bug report, fancy tweaking the text a bit?